src/models/multiChart.js

```diff
@@ -538,6 +538,7 @@ nv.models.multiChart = function() {
         width:      {get: function(){return width;}, set: function(_){width=_;}},
         height:     {get: function(){return height;}, set: function(_){height=_;}},
         showLegend: {get: function(){return showLegend;}, set: function(_){showLegend=_;}},
+        xScale: {get: function(){return x;}, set: function(_){ x = _; xAxis.scale(x); }},
```

Comment:
Isn't this already possible by doing chart.xAxis.scale(...), since xAxis is made available on the main chart object?